jsettlers.mapcreator/src/jsettlers/exceptionhandler/ErrorIcon.java

```diff
@@ -0,0 +1,58 @@
+package jsettlers.exceptionhandler;
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.Polygon;
+import java.awt.RenderingHints;
+
+import javax.swing.Icon;
+
+/**
+ * Icon for error popup
+ * 
+ * @author Andreas Butti
+ */
+public class ErrorIcon implements Icon {
+
+	/**
+	 * Size
+	 */
+	private int size = 120;
```

Comment:
As there is no Button to confirm (like other tools have) I'll simple write "done" to all comments, so I know I have done it. I'll create pull request when I'm finished